lib/storage_tables/service/disk_service.rb

```diff
@@ -36,6 +36,12 @@ def url_for_direct_upload(checksum, expires_in:, content_type:, content_length:)
             purpose: :blob_token
           )
 
+          if url_options.blank?
+            raise ArgumentError,
```

Comment:
Misschien mooier om deze check in de `url_options` method te doen. Je hebt dan niet de checksum beschikbaar, maar ik niet dat dat heel erg is. Als deze geraised wordt is dat niet voor een specifiek document maar voor alle.

Reply:
Done